app/code/community/TIG/MyParcel2014/etc/system.xml

```diff
@@ -309,14 +309,23 @@
                             <show_in_website>1</show_in_website>
                             <show_in_store>1</show_in_store>
                         </retrieve_address_title>
-                        <busy_loading_title> translate="label" module="tig_myparcel">
+                        <busy_loading_title translate="label" module="tig_myparcel">
```

Comment:
Ik zie geen vertaling voor busy_loading_title

Reply:
wat er veranderd is is dat er in eerste instantie `<busy_loading_title>` werd gebruikt. dat moet worden `<busy_loading_title` de `>` is weggehaald